### PR TITLE
feat: Flight Model update

### PR DESCRIPTION
**Pull request by donstim**
## Summary of Changes

This update of the flight model began with (now closed) PR #7258. That PR proposed changes to the landing gear parameters, attempted to improve the airplane flying characteristics in gusts/turbulence, reduced the ground effect, and better match CONF  1+F lift and drag. After pilot comments, there were also attempts to reduce the sensitivity of the pitch control.

The PR was unsuccessful at modeling the roll sensitivity to gusts/ turbulence and in achieving the desired pitch control responsiveness/sensitivity. Further, the manner in which the pitch sensitivity was being adjusted had significant and bad side effects, such as causing drag to increase for further aft cg positions (the opposite of how it should be).

This PR, developed in coordination with some FBW team pilots, achieves the desired pitch control flying characteristics and roll response to gusts/turbulence. It also improves upon the accuracy of previous flight models in matching FCOM/QRH/FlySmart data for pitch and N1 values in all configurations for the conditions represented in the data, stopping distances (braking performance) with manual braking, climb and descent capability, maximum altitude capability, etc. (This is due not only to the accuracy of the flight model, but also the engine model.)

Spoiler drag has been increased a bit to provide an exact match to the FCOM's emergency descent data. The toe brake scalar in the system.cfg file was changed to match stopping distances with the flight model's revised lift/drag characteristics. A very slight change was made to idle thrust at very low speed/N1 to better match the decel condition in the reference document included below. Lastly, changes were made to the VNAV prediction aero parameters to match the new flight model.

There are still a few data-matching issues that are noted below under "Additional Context."
## References

Updated to add EDDW Stopping  distance test results.
Updated again to reflect further autobrake tuning.
[Flight Model Documentation for Sept 2022 Flight Model Update (abbreviated).docx](https://github.com/flybywiresim/a32nx/files/9896721/Flight.Model.Documentation.for.Sept.2022.Flight.Model.Update.abbreviated.docx)

## Additional context

Some remaining issues:
The engine model is in the process of being even more accurately defined. Some areas needing special attention (see the document attached above):
1. Fuel burn. The cruise check data shows the fuel flow to be very good for most of the cruise conditions. It's also pretty good for the ISA+10C climbs, but is lower than the FCOM data for the ISA+20C climbs and for the ISA descents.
2. Thrust/N1. Good match over most of the envelope, but thrust was a bit high for the above ISA maximum altitude checks and the ISA+20C climb data. Min idle thrust looks really good!
3. Autobrake stopping performance. The stopping distance at all weights in both landing configurations for Autobrake Medium and Low settings was significantly better than the predicted distances (see document attached above). (@crocket63)
4. The maximum elevator deflections in both up and down directions were increased in the flight model. This may? have an effect on the ECAM depiction of elevator deflection if the deflections shown are scaled based on the previous max deflection values.

I should also point out that direct law flight characteristics are not addressed at all in this flight model update. 

A sensitivity of -30% for the joystick control in pitch is still recommended (see https://docs.flybywiresim.com/fbw-a32nx/feature-guides/autopilot-fbw/?h=sensitiv#recommended-sensitivity-settings). Roll sensitivity should be linear.

Discord username (if different from GitHub): donbikes#4084

## Testing instructions

Although this has already been evaluated by IRL A320 Neo pilots, experience has shown that a wider review is always necessary, especially from users with different computer systems and controllers. Full flights and multiple landings should be conducted to make sure there aren't any hidden "gotcha's." Handfly as much as possible and enjoy the handling!

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by tracernz — APPROVED** [review]

**Comment by crocket63** [comment]
Any idea about why autobrake would be too good?  As it just follow same decel target, it's either it engages sooner (gear compressed info sooner?), or initial transient goes faster to demanded target. Can't really think of other reasons?

**Comment by donstim** [comment]
> Any idea about why autobrake would be too good? As it just follow same decel target, it's either it engages sooner (gear compressed info sooner?), or initial transient goes faster to demanded target. Can't really think of other reasons?

I don't know the reason. I'm only reporting the results of my testing. It could either be engaging to quickly or the ramp up is too quick. It did reach and hold the intended target decel rate of 3 m/s squared in the one case I checked. It doesn't take much of a difference in the ramp up to cause the differences I saw since it's at the highest speed portion of the stop. The overall average decel rate should have been around 2.42 m/s squared to match the FlySmart distance, but it was more like 2.54 m/s squared.

By the way, this is not new. I reported something similar back in May or June.

File: src/fmgc/src/guidance/vnav/FlightModel.ts [hunk 1]
```diff
@@ -52,25 +52,25 @@ export class FlightModel {
...
-            baseDrag = (-0.0207 * Cl ** 5) + (0.0764 * Cl ** 4) - (0.0813 * Cl ** 3) + (0.0912 * Cl ** 2) - (0.0285 * Cl) + 0.0337;
+            baseDrag = (0.0385 * Cl ** 3) + (0.004 * Cl ** 2) + (0.0044 * Cl) + 0.0249;
             break;
         case FlapConf.CONF_2:
-            baseDrag = (0.0066 * Cl ** 5) - (0.0271 * Cl ** 4) + (0.0615 * Cl ** 3) - (0.0187 * Cl ** 2) + (0.0035 * Cl) + 0.0538;
+            baseDrag = (0.0242 * Cl ** 3) + (0.0013 * Cl ** 2) + (0.003 * Cl) + 0.0485;
             break;
         case FlapConf.CONF_3:
-            baseDrag = (0.0768 * Cl ** 5) - (0.3979 * Cl ** 4) + (0.8252 * Cl ** 3) - (0.7951 * Cl ** 2) + (0.3851 * Cl) - 0.0107;
+            baseDrag = (0.0221 * Cl ** 3) - (0.0027 * Cl ** 2) + (0.0033 * Cl) - 0.0584;
```
Comment (BlueberryKing) [inline src/fmgc/src/guidance/vnav/FlightModel.ts:64]: Hi, did a quick test with VNAV and predictions were off. Is it possible this minus sign is incorrect? The curve is very similar to the old one if the sign is switched,
```suggestion
            baseDrag = (0.0221 * Cl ** 3) - (0.0027 * Cl ** 2) + (0.0033 * Cl) + 0.0584;
```

**Comment by donstim** [comment]
Would be nice to get the autobrake stopping distances re-tuned. At least one person has noticed this...https://discord.com/channels/738864299392630914/738864300818432023/1034254014730215494

**Comment by crocket63** [comment]
> > Any idea about why autobrake would be too good? As it just follow same decel target, it's either it engages sooner (gear compressed info sooner?), or initial transient goes faster to demanded target. Can't really think of other reasons?
> 
> I don't know the reason. I'm only reporting the results of my testing. It could either be engaging to quickly or the ramp up is too quick. It did reach and hold the intended target decel rate of 3 m/s squared in the one case I checked. It doesn't take much of a difference in the ramp up to cause the differences I saw since it's at the highest speed portion of the stop. The overall average decel rate should have been around 2.42 m/s squared to match the FlySmart distance, but it was more like 2.54 m/s squared.
> 
> By the way, this is not new. I reported something similar back in May or June.

This should solve the issue #7568. Hopefully we don't end up too far from expected values, else we'll have to change the decel profile itself.

**Comment by donstim** [comment]
Added changes from #7570. Need to check to see if this affects the flight model tuning.

**Comment by dragold** [comment]
Category: QA Tester
Name: Mr.Blank#6197
Date of testing MM/DD/YYYY:11/06/2022

PR Tested: 7556
Tier of testing:1

Testing technique: tryouts
Test Instructions:
experience has shown that a wider review is always necessary, especially from users with different computer systems and controllers. Full flights and multiple landings should be conducted to make sure there aren't any hidden "gotcha's." Handfly as much as possible and enjoy the handling!

Some remaining issues:
Fuel burn. The cruise check data shows the fuel flow to be very good for most of the cruise conditions. It's also pretty good for the ISA+10C climbs, but is lower than the FCOM data for the ISA+20C climbs and for the ISA descents.
Thrust/N1. Good match over most of the envelope, but thrust was a bit high for the above ISA maximum altitude checks and the ISA+20C climb data. Min idle thrust looks really good!
Autobrake stopping performance. The stopping distance at all weights in both landing configurations for Autobrake Medium and Low settings was significantly better than the predicted distances (see document attached above). ([@crocket63](https://github.com/crocket63))
The maximum elevator deflections in both up and down directions were increased in the flight model. This may? have an effect on the ECAM depiction of elevator deflection if the deflections shown are scaled based on the previous max deflection values.
I should also point out that direct law flight characteristics are not addressed at all in this flight model update.
A sensitivity of -30% for the joystick control in pitch is still recommended (see https://docs.flybywiresim.com/fbw-a32nx/feature-guides/autopilot-fbw/?h=sensitiv#recommended-sensitivity-settings). Roll sensitivity should be linear.

Test Done.
All as per insturction 
Negatives: NIL
Testing Results: PASS 
Conclusions: results were of normal outputs

**Pull request merged**